From d351f170ace0dad53c85817e6a82032645efd742 Mon Sep 17 00:00:00 2001
From: Alexander Smirnov <asmirnov@ilbers.de>
Date: Sat, 13 Dec 2014 09:40:43 +0100
Subject: [PATCH 1/2] exynos5250: remove second CPU from device tree

Remove second CPU from exynos5250 device tree due to Mango supports only
single core per partition.

Signed-off-by: Alexander Smirnov <asmirnov@ilbers.de>
---
 arch/arm/boot/dts/exynos5250.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/boot/dts/exynos5250.dtsi b/arch/arm/boot/dts/exynos5250.dtsi
index d55c1a2..03bcf5f 100644
--- a/arch/arm/boot/dts/exynos5250.dtsi
+++ b/arch/arm/boot/dts/exynos5250.dtsi
@@ -64,12 +64,6 @@
 			reg = <0>;
 			clock-frequency = <1700000000>;
 		};
-		cpu@1 {
-			device_type = "cpu";
-			compatible = "arm,cortex-a15";
-			reg = <1>;
-			clock-frequency = <1700000000>;
-		};
 	};
 
 	sysram@02020000 {
-- 
2.1.4

